test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
```

Comment:
What is a `const` anyway? A constant reference to a variable or a reference to a constant variable?